tests/functional/ionIonBeam/ion_ion_beam1d.py

```diff
@@ -143,52 +151,85 @@ def growth_b_right_hand(run_path):
     # the factor "+1" is because we remove the DC component, so the value
     # given by argmax has also to miss this value
     omegas = np.fabs(np.fft.fft(damped_mode).real)
-    omega = 0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[-1]
+    omega = 0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[imax-1 -int(d/dt)]
 
-    print(omegas[0:6])
-    print(omegas[1:omegas.size//2].argmax())
-    print(2*np.pi/times[-1])
-    print(0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[-1])
 
     return times, first_mode, popt[0], popt[1], damped_mode, omega
 
-
 def main():
-    from pybindlibs.cpp import mpi_rank
 
+    from pybindlibs.cpp import mpi_rank
+    d = 10 # The exponential fit is performed on [0, times[imax] - d]
     config()
     simulator = Simulator(gv.sim)
     simulator.initialize()
+    
     simulator.run()
-
+    
     if mpi_rank() == 0:
+    
 
+    
         times, first_mode, ampl, gamma, damped_mode, omega \
-                = growth_b_right_hand(os.path.join(os.curdir, "ion_ion_beam1d"))
-
-        fig, (ax1, ax2) = plt.subplots(2, 1)
-
-        ax1.set_title("Right Hand Resonant mode (Beam instability)")
-        ax1.stem(times, first_mode, linefmt='-k', basefmt=' ', use_line_collection=True)
-        ax1.plot(times, yaebx(times, ampl, gamma), color='r', linestyle='-', marker='')
-        ax1.text(0.04, 0.80, "From Gary et al., 1985 (ApJ : 10.1086/162797)", transform=ax1.transAxes)
-        ax1.set_ylabel("Most unstable mode")
-        ax1.set_title("Right Hand Resonant mode (Beam instability)")
-        ax1.text(0.30, 0.50, "gamma = {:5.3f}... expected 0.09".format(gamma), transform=ax1.transAxes)
-
-        ax2.plot(times, damped_mode, color='g', linestyle='', marker='o')
-        ax2.set_xlabel("Time")
-        ax2.set_ylabel("Real mode")
-        ax2.text(0.48, 0.30, "~ 3 periods until t=50", transform=ax2.transAxes)
-        ax2.text(0.40, 0.20, "omega (real) = {:5.3f}... expected 0.19".format(omega), transform=ax2.transAxes)
-
+                = growth_b_right_hand(os.path.join(os.curdir, "ion_ion_beam1d"),d)
+
+        dt = times[1]-times[0]
```

Comment:
Should be replaced by dt_stamps = 0.1 to be consistant with line 118's proposition